packages/jokul/src/components/popover/integration/popover.spec.ts

```diff
@@ -20,7 +20,10 @@ test("renders correctly", async () => {
     await helper.open();
 
     await helper.snapshots({
-        before: () => helper.clickElement('[aria-haspopup="menu"]'),
+        before: () =>
+            helper
+                .clickElement('[aria-haspopup="menu"]')
+                .then(() => new Promise((resolve) => setTimeout(resolve, 200))),
```

Comment:
Dette er en fin case for å ha tokens for denne typer verdier i komponentene. Hadde vært fint å kunne bruke noe á la `tooltip.animation.duration.entry` i stedet for et magisk tall (riktig tall er 250, forresten 😅). Ikke noe du trenger å gjøre her altså, bare en observasjon!

Reply:
Det finnes jo en motion.ts fil i den gamle core-pakka, men fant den ikke i den nye 😬